content-api/README.md

```diff
@@ -57,8 +57,7 @@ the first approver:
     `active` of type boolean, and value true.
     - Click *Save*
 
-The person with this email address will be able to perform
-all API operations as an Approver.
+The person with this email address will be able to perform all API operations as an Approver.
```

Comment:
Need to port this to the new location in docs/content-api.md

Reply:
Oh, no change here. I had removed the line change and mistakenly committed it here. 
Going to close this PR in favor of breaking up the minor updates / bug I was targeting!